packages/rainbow-button/package.json

```diff
@@ -39,11 +39,8 @@
   "peerDependencies": {
     "react": ">=17",
     "react-dom": ">=17",
-    "viem": "~0.3.19 || ^1.0.0",
-    "wagmi": "~1.0.1 || ~1.1.0 || ~1.2.0 || ~1.3.0 || ~1.4.0"
-  },
-  "devDependencies": {
-    "@wagmi/core": "^1.4.3"
+    "viem": "2.0.0",
+    "wagmi": "2.0.0-rc.2"
```

Comment:
This looks problematic; we still need these as peers I believe

Reply:
Yep that's right. Going to update it on all packages / examples like we have here:
- https://github.com/rainbow-me/rainbowkit/blob/1d602e87d8686dbe1d83e6ba64c38022f99c7681/examples/with-next-app/package.json#L16

- https://github.com/rainbow-me/rainbowkit/blob/1d602e87d8686dbe1d83e6ba64c38022f99c7681/packages/rainbow-button/package.json#L42

Reply:
Seems like wagmi requires their `viem` package to be `viem@2.0.0-rc.3`. Here are the [docs](https://rc.wagmi.sh/react/getting-started#manual-installation)